File: Java/xProjectDemoJava/app/src/main/java/com/example/xprojectdemojava/MainActivity.java
package com.example.xprojectdemojava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    TextView txtuser,txtcontra;
    Button btnEntrar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txtuser=(EditText) findViewById(R.id.txtuser);
        txtcontra=(EditText) findViewById(R.id.txtcontra);
        btnEntrar=(Button) findViewById(R.id.btnEntrar);

        btnEntrar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                    Intent intent=new Intent(MainActivity.this,MenuInicial.class);
                    startActivity(intent);
                    finish();
            }
        });

    }
}

File: CnfrmFact/app/src/main/java/com/example/cnfrmfact/MainActivity.kt
package com.example.cnfrmfact

import android.content.Intent
import android.content.SharedPreferences
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import org.json.JSONException
import org.json.JSONObject

class MainActivity : AppCompatActivity() {
    var mApp = GlobalsVar()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnEntrar.setOnClickListener {

            GetuserLoggin()


        }
    }//Fin Oncreate

    //JsonObject----------------------
    private fun GetuserLoggin() {
        //si la rsspuesta json del loggin es correcta asignamos el user a PREF_USER
        val user = txtuser
        var contra = txtcontra

        if(user.text.toString()=="admin" && contra.text.toString()=="admin")
        {
            val sharedPref: SharedPreferences = getSharedPreferences(mApp.PREF_USER, mApp.PRIVATE_MODE)
            val editor = sharedPref.edit()
            editor.putString(mApp.PREF_USER, user.text.toString())
            editor.commit()

            val inte= Intent(this,Main2ActivityFacturaScan::class.java)
            startActivity(inte)
            finish()

        }
        else
        {
            Toast.makeText(this@MainActivity, "Usuario/Contra incorrecta ", Toast.LENGTH_SHORT).show()
        }





    }///Fin JsonObject---------------
}
